src/app/services/views/serviceConfig.ejs

```diff
@@ -395,12 +395,26 @@
                 <hr class="govuk-section-break govuk-section-break--visible">
 
                 <div class="govuk-button-group auto-scroll-dest govuk-!-margin-top-3">
-                    <button type="submit" class="govuk-button" data-module="govuk-button" id="continue-button">Continue</button>
+                    <button type="submit" onclick="sanitize()" class="govuk-button" data-module="govuk-button" id="continue-button">Continue</button>
                     <a href="<%= locals.backLink %>" class="govuk-button govuk-button--secondary">Cancel</a>
                 </div>
             </form>
         </div>
     </div>
 </div>
 <script src="/assets/javascript/niceware.js" async></script>
+<script>
+   //this is to correct an encoding issue found with < charcter not sending full strings to server
```

Comment:
Remove the unnecessary comments please function code is self explanatory, can you check for null if there is one.